app/src/main/java/org/simple/clinic/scheduleappointment/ScheduleAppointmentModel.kt

```diff
@@ -3,19 +3,17 @@ package org.simple.clinic.scheduleappointment
 import android.os.Parcelable
 import kotlinx.android.parcel.Parcelize
 import org.simple.clinic.facility.Facility
+import org.simple.clinic.newentry.ButtonState
 import org.simple.clinic.overdue.PotentialAppointmentDate
 import org.simple.clinic.overdue.TimeToAppointment
 import org.simple.clinic.teleconsultlog.teleconsultrecord.TeleconsultRecord
 import org.simple.clinic.util.UserClock
 import java.util.UUID
-import org.simple.clinic.newentry.ButtonState as ButtonState
 import org.simple.clinic.scheduleappointment.ButtonState as NextButtonState
 
 @Parcelize
 data class ScheduleAppointmentModel(
     val patientUuid: UUID,
-    val potentialAppointmentDates: List<PotentialAppointmentDate>,
```

Comment:
I think we can just drop this commit because we are moving these variables as computed properties in the next commit. WDYT?